system_baseline/views/v0.py

```diff
@@ -131,23 +157,52 @@ def delete_baselines_by_ids(baseline_ids):
     return "OK"
 
 
+def _create_ordering(order_by, order_how, query):
+    """
+    helper method to set ordering on query. `order_by` and `order_how` are
+    guaranteed to be populated as "DESC" or "ASC" via the openapi definition.
+    """
+    if order_by == "display_name":
+        if order_how == "DESC":
+            query = query.order_by(SystemBaseline.display_name.desc())
+        elif order_how == "ASC":
+            query = query.order_by(SystemBaseline.display_name.asc())
+    elif order_by == "created_on":
+        if order_how == "DESC":
+            query = query.order_by(SystemBaseline.created_on.desc())
+        elif order_how == "ASC":
+            query = query.order_by(SystemBaseline.created_on.asc())
+
+    return query
+
+
 @metrics.baseline_fetch_all_requests.time()
 @metrics.api_exceptions.count_exceptions()
-def get_baselines(limit, offset):
+def get_baselines(limit, offset, order_by, order_how, display_name=None):
     """
     return a list of baselines given their ID
     """
     account_number = view_helpers.get_account_number(request)
     query = SystemBaseline.query.filter(SystemBaseline.account == account_number)
 
+    if display_name:
+        query = query.filter(SystemBaseline.display_name == display_name)
```

Comment:
Is this an exact match search? How would I find all baselines with X in the name?

Reply:
I altered the search to be "contains" instead of exact match